docs/tutorials/neutron-diffraction.ipynb

```diff
@@ -331,12 +391,27 @@
     "theta = sc.Variable(['scattering_angle'],\n",
     "                    unit=sc.units.rad,\n",
     "                    values=np.linspace(0.0, np.pi/2, num=16))\n",
-    "focussed = sc.groupby(\n",
-    "    dspacing_events,\n",
-    "    'scattering_angle',\n",
-    "    bins=theta).flatten('spectrum')\n",
-    "focussed = sc.histogram(focussed, dspacing_bins)\n",
-    "normalized = focussed['sample'] / focussed['vanadium']"
+    "\n",
+    "# Option 1: Realign, then group dataset, end result event data\n",
+    "dspacing = sc.Dataset({\n",
+    "    'sample':sc.realign(dspacing_sample, {'d-spacing':dspacing_bins}),\n",
+    "    'vanadium':sc.realign(dspacing_vanadium, {'d-spacing':dspacing_bins})\n",
+    "})\n",
+    "focussed = sc.groupby(dspacing, 'theta', bins=theta).sum('spectrum')\n",
+    "normalized = focussed['sample'] / sc.histogram(focussed['vanadium'])\n",
+    "\n",
```

Comment:
👍 excellent. might want to ensure the point is made here that your output is still in event mode.